src/simulate.jl

```diff
@@ -31,12 +31,12 @@ function simulate(series::Vector{T}, gas::Model{D, T}, H::Int, S::Int;
     parameter_scenarios = Array{T, 3}(undef, H, n_params, S)
     for scenario in 1:S
         # Notice that we know the parameter time_varying parameters for T + 1 
-        # So the last initial_params is already a part of the future simulation
-        # And we must take the 
+        # So the last initial_params is already a part of the future simulation.
+        # When we simulate 60 steps ahead the last observation is one too much for us.
```

Comment:
This comment isn't too clear to me, maybe elaborate a bit here? Why 60 steps? What does it mean one too much?